### ISSUE TITLE
AndroidTV Channels - Greyed cover-thumbs when starting the system

### ISSUE THREAD

**Issue by sorryusernameisalreadytaken** [xbmc/xbmc#17431]
### Describe the bug
Since Android TV 8 and Kodi v18 it is possible to setup a smartplaylist for the AndroidTV channel feature. I did that and had the issue that all thumb-coversa are greyed-out after init-start. I need to start kodi once to get the thumbs.

It seems I am not alone with this issue. Some other members in the german board here had the same issue:
https://www.kodinerds.net/index.php/Thread/62103-How-To-Zus%C3%A4tzliche-Kodi-Channels-in-Android-8-Leanback-Homescreen-anlegen-mit-Ko/?postID=517957#post517957

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

After a fresh start of the system the thumb-covers should be visible if they are already present.

## Actual Behavior

After a fresh system start there are no cover. Just grey cover thumbs.

## Possible Fix

It seems that this is a issue with shieldTV 2015 and 2017 model. A user in the mentioned german kodi board tell that had no issues with the nvidia shield-tv 2019 edition. If i am not wrong the 2019 is arm and the 2015/2017 are arm64, or?

### To Reproduce

1. setup smartplaylist with content
2. be sure that kodi had already load the thumbs
3. add the channel to your android-tv launcher
4. restart your device

### Debuglog

The debuglog can be found here:
[logcat_kodi_grey_thumbs.txt](https://github.com/xbmc/xbmc/files/4281903/logcat_kodi_grey_thumbs.txt)

### Screenshots 
Here are some links or screenshots to help explain the problem:

![kodi_Androidtv_launcher_problem](https://user-images.githubusercontent.com/12913436/75786619-de3f6c80-5d65-11ea-98b7-bb619398e27c.png)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

I see in the log:

> 03-03 15:52:06.714  6373  9876 E org.xbmc.kodi: No implementation found for boolean org.xbmc.kodi.XBMCFile._open(java.lang.String) (tried Java_org_xbmc_kodi_XBMCFile__1open and Java_org_xbmc_kodi_XBMCFile__1open__Ljava_lang_String_2)

### Your Environment
Used Operating system:

 - [X] Android

 - Operating system version/name:
 - Kodi version: Kodi 18.5 Git:20191116-37f51f6e63

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by Dnkhatri** [comment]
I think @koying fixed this for mrmc by running portion of the mrmc as a service

**Comment by sorryusernameisalreadytaken** [comment]
> 
> 
> I think @koying fixed this for mrmc by running portion of the mrmc as a service

Does that mean is needs a separate apk? Netflix for example do a nice job and I think it does not need a special service. The only different thing is that it seems to fetch the covers from an online source. When I disconnect my shieldTV is have the same behavior than this bug here: greyed covers.

**Comment by koying** [comment]
No. An android service is just a part of an application (same APK) running in background on a different thread and with no UI.
Most of android apps have some kind of service, and definitely all that send you notifications.

**Comment by sorryusernameisalreadytaken** [comment]
okay, that make sense and it explains why I need to start kodi once. If we had just a opportunity to start kodi with no gui and put this into a autostart script/app....

**Comment by koying** [comment]
Well, you should be able to autostart Kodi *with* gui.
Same as https://github.com/koying/SPMC/wiki/How-to-autostart-SPMC-on-boot

The code is still there, but no clue if it actually still works

**Comment by Dnkhatri** [comment]
> Well, you should be able to autostart Kodi _with_ gui.
> Same as https://github.com/koying/SPMC/wiki/How-to-autostart-SPMC-on-boot
> 
> The code is still there, but no clue if it actually still works

not according to you
https://forum.kodi.tv/showthread.php?tid=309042

**Comment by koying** [comment]
Post is from March 2017, I backported in May https://github.com/xbmc/xbmc/pull/12169
